core/tauri/src/manager.rs

```diff
@@ -626,7 +626,21 @@ impl<R: Runtime> WindowManager<R> {
             response = response.header(k, v);
           }
 
-          let mime_type = MimeType::parse(&data, &path);
+          let mime_type = {
+            use once_cell::sync::Lazy;
+            static MIME_TYPES_CACHE: Lazy<Arc<Mutex<HashMap<String, String>>>> =
```

Comment:
I'd try moving this data structure to the Manager struct

Reply:
the `InnerWindowManager` struct

Reply:
yeah (I need to cleanup that manager struct, the WindowManager has too many stuff lol)

Reply:
I think I could just use the `StateManager` to manage it, no?

Reply:
yeah that would be ok too